integrations/openshift.md

```diff
@@ -39,585 +39,77 @@ Here's a preview of the Kubernetes Pods dashboard:
 
 {% include image.md src="images/db_kubernetes_pods.png" width="80" %}
 
+## Use OpenShift To Install The Kubernetes Integration
 
-## Kubernetes Quick Install Using Helm
-1. Ensure that you have installed [helm](https://helm.sh/docs/intro/).
-2. Add the Wavefront helm repo:{% raw %}
-```
-helm repo add wavefront https://wavefronthq.github.io/helm/
-helm repo update
-```
-{% endraw %}
-3. To deploy the Wavefront Collector and Wavefront Proxy:
-
-    Using helm 2:{% raw %}
-    ```
-    helm install wavefront/wavefront --name wavefront --set wavefront.url=https://YOUR_CLUSTER.wavefront.com --set wavefront.token=YOUR_API_TOKEN --set clusterName=<YOUR_CLUSTER_NAME> --namespace wavefront
```

Comment:
...--set wavefront.url=https://YOUR_WF_INSTANCE.wavefront.com 
RK>>I'd call it Wavefront instance, not your_cluster. The doc always uses instance, we make no guarantees about the config. 
RK>>Maybe also YOUR_WF_API_TOKEN 
??